src/nrc/api/tasks.py

```diff
@@ -11,7 +13,19 @@
 logger = logging.getLogger(__name__)
 
 
-@app.task(ignore_result=True)
+class NotificationException(Exception):
+    pass
+
+
+@app.task(
+    autoretry_for=(
+        NotificationException,
+        requests.exceptions.RequestException,
+    ),
+    max_retries=settings.CELERY_MAX_RETRIES,
+    retry_backoff=settings.CELERY_RETRY_BACKOFF,
+    retry_backoff_max=settings.CELERY_RETRY_BACKOFF_MAX,
```

Comment:
please do not use `CELERY_` prefixed setting names for this, as this creates the illusion that these are settings implemented in Celery itself (https://docs.celeryq.dev/en/stable/userguide/configuration.html)